include/mbedtls/mbedtls_config.h

```diff
@@ -3609,13 +3610,15 @@
 //#define MBEDTLS_PLATFORM_SETBUF_MACRO      setbuf /**< Default setbuf macro to use, can be undefined */
 //#define MBEDTLS_PLATFORM_TIME_MACRO            time /**< Default time macro to use, can be undefined. MBEDTLS_HAVE_TIME must be enabled */
 //#define MBEDTLS_PLATFORM_TIME_TYPE_MACRO       time_t /**< Default time macro to use, can be undefined. MBEDTLS_HAVE_TIME must be enabled */
+//#define MBEDTLS_PLATFORM_MS_TIME_TYPE_MACRO   signed long long /**< Default millionseconds time macro to use, can be undefined. MBEDTLS_HAVE_TIME must be enabled. It MUST be signed 64bit integer at least */
```

Comment:
should this match the definition in `platform_time.h`?

Reply:
I just revert it to signed long long. I found build fail with`int64_t` in #6788

Reply:
Can you provide a link to the failing build so we can see why? I still think having specifically-sized types is the best way to do this kind of thing

Reply:
Sorry, I just got your message. And I have force push the last commit.

It is print relative issues. For different compiler, the print fmt string are different. It might be "ld" or "lld" .  Then it reports fmt error. 

I found a stand solution for c99. As https://www.open-std.org/jtc1/sc22/wg14/www/docs/n1256.pdf , section 7.8, `PRIdN` are standard macros for the fmt string. I think we can use that. 

`MBEDTLS_PRINTF_LONGLONG`  in `debug.h` can be replace with PRId64 .

Reply:
> Can you provide a link to the failing build so we can see why? I still think having specifically-sized types is the best way to do this kind of thing

I found it in my local. In https://github.com/Mbed-TLS/mbedtls/blob/18d2fffaee572d35536462a6d7bc4f36244c9dc0/library/ssl_tls13_server.c#L170 , It reports format string error.